Library/Formula/dcadec.rb

```diff
@@ -0,0 +1,20 @@
+class Dcadec < Formula
+  desc "DTS Coherent Acoustics decoder with support for HD extensions"
+  homepage "https://github.com/foo86/dcadec"
+  url "https://github.com/foo86/dcadec/archive/v0.1.0.tar.gz"
+  sha256 "52a68a63f79ae50e5e5fd9c631a715e9feeed5d2b4a8442f0b9208692012a43f"
+  head "https://github.com/foo86/dcadec.git"
+
+  def install
+    system "make", "all"
+    unless File.exist?("test/samples/README")
```

Comment:
Nope, `--HEAD` is still unlocked but will share the repository. Homebrew will just check it out to the right revision.